Socket/client/chatapp/src/Component/Common/Message/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useContext } from "react";
+import { Avatar, Badge } from "antd";
+
+import { UserOutlined } from "@ant-design/icons";
+// @ts-ignore
+import { EditSquareLightIcon, DeleteLightIcon } from "@iconbox/iconly";
+
+import Context from "../../../Context";
+import { StyledAvatarWrapper } from "../User/style";
+
+import {
+  StyledMessageItem,
+  StyledMessageText,
+  StyledIconsWrapper,
+} from "./style";
+
+interface MessageProp {
+  msg: {
+    user: {
+      id: string;
+      name?: string;
+    };
+    message: string;
+  };
+  key: number;
+}
```

Comment:
- use name `Props` always for component props
- change `msg` to `message`
- use a global `types` file and add `Message` type there to use it everywhere:
```
  export type Message {
     user: {
        id: string;
        name?: string;
     };
     message: string;
  }
```